cmd/guaccollect/cmd/s3.go

```diff
@@ -49,8 +52,12 @@ $ export AWS_SECRET_ACCESS_KEY=zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG
 
 Ingest:
 
-$ guacone collect s3 --s3-url https://play.min.io --s3-bucket guac-test
-$ guacone collect s3 --s3-url play.min.io --s3-bucket guac-test --s3-item alpine-cyclonedx.json
+$ guaccollect s3 --s3-url https://play.min.io --s3-bucket guac-test
+$ guaccollect s3 --s3-url play.min.io --s3-bucket guac-test --s3-item alpine-cyclonedx.json
+
+Ingest from AWS using default url:
+
+$ guaccollect s3 --s3-bucket guac-test --s3-region eu-north-1
```

Comment:
nit: can we add a `s3path` example here also?

Reply:
+1

Reply:
thanks for the heads, added example!